runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingConfig.java

```diff
@@ -143,4 +143,15 @@ default Tab activeTab()
 		hidden = true
 	)
 	void setActiveTab(Tab t);
+
+	@ConfigItem(
+		keyName = "notifyOnDisease",
+		name = "Notify On Disease",
+		description = "Always send a notification when disease is detected via Geomancy.",
```

Comment:
this should not be specific to geomancy

Reply:
This would require making some semi-large changes to FarmingTracker to support disease notifications, do you want me to add that as well?

Reply:
never mind, added that here: e44d73d6d8d8839eee33787cdcaa050f4156e00b